display.php

```diff
@@ -26,7 +26,8 @@ function cablecast_content_display($content){
         $trt = get_post_meta($post->ID, 'cablecast_show_trt', true);
         $show_content =  "<div>";
         if (is_single()) {
-          $show_content .= "[video src=\"$vod_url\"]";
+          $vod_poster = get_the_post_thumbnail_url();
+          $show_content .= "[video src=\"$vod_url\" poster=\"$vod_poster\" autoplay=\"true\"]";
```

Comment:
I'm on the fence about the autoplay mostly because of my personal disdain for it ;) It also doesn't work on most mobile browsers anyway.

The poster is great though so 👍 

The intent for these methods was to be pretty bar bones as I figured most people would just grab the data in there themes, though I'm not 100% sure how normal wordpressers do designs.

Reply:
True! I like having the autoplay param in there so it's obvious it can be set to true or false.